core/src/main/java/brooklyn/catalog/internal/CatalogUtils.java

```diff
@@ -227,4 +228,13 @@ public static String getVersionedId(String id, String version) {
         }
     }
 
+    @Beta
+    public static EntitySpec<?> createEntitySpec(ManagementContext mgmt, CatalogItem<?, ?> catalogItem) {
```

Comment:
good spot.  i completely agree -- i had missed that config etc would be ignored (d'oh).

have moved to `CatalogTestUtils.createEssentialEntitySpec`.